partner_company_type/README.rst

```diff
@@ -14,16 +14,16 @@ Partner Company Type
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
-    :target: https://github.com/OCA/partner-contact/tree/15.0/partner_company_type
+    :target: https://github.com/OCA/partner-contact/tree/16.0/partner_company_type
     :alt: OCA/partner-contact
 .. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
     :target: https://translation.odoo-community.org/projects/partner-contact-15-0/partner-contact-15-0-partner_company_type
```

Comment:
:target: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_company_type

Reply:
No need to change. Bot will do